frontend/public/tutorial.md

```diff
@@ -1,12 +1,64 @@
 ## Getting Started
 
-- In `main.tsx`, update the options passed `SquidContextProvider` with you `appId` and `squidDeveloperId`.
-- Run the `Create .env file` script in (found in the [Squid Console](https://console.squid.cloud)) in the `backend` directory.
-- Run `squid start` in the backend directory.
+Welcome to your first Squid project! Through a few simple steps, you will add functionality to a full-stack to-do application and learn about some of the features of Squid.
 
-## Creating a Todo
+1. In the [Squid Cloud Console](https://console.squid.cloud) create a new app called `todo`.
 
-Querying for todos. In `App.tsx` add the following:
+2. To initialize Squid in a client app, you must provide some details about your Squid application. In the `main.tsx` file of the frontend, update the options passed to `SquidContextProvider` with your `appId` and `squidDeveloperId`. You can find these values in the [Squid Cloud Console](https://console.squid.cloud):
+
+```typescript
+...
+  <SquidContextProvider
+    options={{
+      appId: 'YOUR_APP_ID', // Update the App ID
+      region: 'us-east-1.aws',
+      environmentId: 'dev',
+      squidDeveloperId: 'YOUR_SQUID_DEVELOPER_ID', // Update the developer ID
+    }}
+  >
+...  
+```
+
+3. In the [Squid Cloud Console](https://console.squid.cloud), scroll to the **Backend project** section and click **Create .env vars**. 
+
+4. Open a new terminal window. You should now have two terminal windows open: one running the frontend of this app as instructed in the README, and another window that will run the backend. Copy the command to install the Squid CLI, and then run it in the terminal:
+
+```bash
+npm install -g @squidcloud/cli
+```
+
+The Squid CLI lets you run your Squid projects locally during development and testing, and then deploy your backend when ready.
+
+5. Navigate to the `backend` directory:
+
+```bash
+cd backend
+```
+
+6. Copy the command to create the `.env` file that's shown in the **Backend project** section of the Squid Cloud Console, and then run it in the `backend` directory. The script has the following format:
```

Comment:
nit: We say "command" and then say "script". I'd just stick to one term in an effort to eliminate any possible confusion